hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/RestoreJob.java

```diff
@@ -30,6 +30,9 @@
 
 @InterfaceAudience.Private
 public interface RestoreJob extends Configurable {
+
+  String KEEP_ORIGINAL_SPLITS_OPT = "keep_original_splits";
```

Comment:
I missed this before, but it's customary to include `hbase.` as a prefix on configuration keys that are used by HBase. I suggest `hbase.backup.restorejob.keep_original_splits`.